assets/l10n/intl_en.arb

```diff
@@ -2711,5 +2711,8 @@
         "placeholders": {
             "directory": {}
         }
-    }
+    },
+    "notInAChatYet": "You're not in a chat yet",
+    "chooseAChatOrHit": "Choose a chat or hit",
```

Comment:
Dont split the sentence anymore
Some languages won't work
Please use markdown or special symbol in string, then use TextSpan to replace it by Widget

Reply:
It is what I wanted to do initially. @nqhhdev wdyt ?